minidump-common/src/format.rs

```diff
@@ -673,11 +673,15 @@ pub enum ExceptionCodeMac {
     SIMULATED = 0x43507378,
 }
 
-/// Valid bits in a `context_flags` for `ContextFlagsCpu`
+/// Valid bits in a `context_flags` for [`ContextFlagsCpu`]
```

Comment:
I might reword this to be "Valid bits in [`ContextFlagsCpu::context_flags`]"

Reply:
I can't actually see that target when building the docs, I haven't used this API yet so not sure what's is supposed to point at.